frontend/benefit/applicant/src/components/applications/pageContent/DecisionSummary.tsx

```diff
@@ -48,6 +50,16 @@ const DecisionSummary = ({ application }: Props): JSX.Element => {
       alteration.alterationType === ALTERATION_TYPE.TERMINATION
   );
 
+  const sortedAlterations = application.alterations?.sort((a, b) => {
```

Comment:
`return new Date(a.endDate) - new Date(b.endDate)` is enough in JS 😉

Reply:
Ah, right, that's true. I'll simplify it to that.